zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/provider/DefaultZaakUrlProvider.kt

```diff
@@ -28,6 +31,13 @@ class DefaultZaakUrlProvider(
 
     @Throws(ZaakInstanceLinkNotFoundException::class)
     override fun getZaakUrl(documentId: UUID): URI {
-        return zaakInstanceLinkService.getByDocumentId(documentId).zaakInstanceUrl
+        return withLoggingContext("documentId" to documentId.toString()) {
```

Comment:
Since this is the identifier of a valtimo entity, it should reference the document class. So either Document::class if it is on the classpath or the classname as a string if not.